old_configs/v0.18.4.yaml

```diff
@@ -0,0 +1,33 @@
+# startup_input: "!"
+stages:
+  - db:   
+      data_dir: ~/.postgres
+
+  - master:
+      pre:
+        - sh: make -C proto build
+        - sh: make -C master build
+        - sh: make -C tools prep-root
+        - sh: mkdir -p /tmp/determined-cp
+      config_file:
+        db:
+          host: localhost
+          port: 5432
+          password: postgres
+          user: postgres
+          name: determined
+        checkpoint_storage:
+          type: shared_fs
+          host_path: /tmp/determined-cp
+        log:
+          level: debug  
+        root: tools/build
+  
+  - agent:
+      pre:
+        - sh: make -C agent build
+      config_file:
+        master_host: 127.0.0.1
+        master_port: 8080
+        container_master_host: $DOCKER_LOCALHOST
+        
```

Comment:
github shows that there's a missing newline at the end of this file.  Can you configure your editor to not leave empty lines at the end of the file, then fix this file?

"every line ends with a newline" is pretty standard in unix.